src/views/virtualmachines/actions/hooks/useVirtualMachineActionsProvider.ts

```diff
@@ -60,14 +65,18 @@ const useVirtualMachineActionsProvider: UseVirtualMachineActionsProvider = (
       pauseOrUnpause,
       VirtualMachineActionFactory.clone(vm, createModal),
       VirtualMachineActionFactory.snapshot(vm, createModal),
-      migrateOrCancelMigration,
+      {
```

Comment:
all the rest are nice funcs, why entering this obj like this? please make it also a func